src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -151,6 +151,17 @@ public string UniqueId
             }
         }
 
+        /// <summary>
+        /// Get the Element Pinned status
+        /// </summary>
+        public bool IsPinned
+        {
+            get 
+            { 
+                return this.InternalElement.Pinned; 
```

Comment:
might need a `null` check here, there's potential for `InternalElement` to be null.
```suggestion
                return this.InternalElement?.Pinned; 
```